doc/filters/country_timezones.rst

```diff
@@ -4,8 +4,8 @@
 .. versionadded:: 2.12
     The ``country_timezones`` filter was added in Twig 2.12.
 
-The ``country_timezones`` filter returns the names of the countries associated
-with a given timezone:
+The ``country_timezones`` filter returns the names of the timezones associated
+with a given country code:
```

Comment:
should this actually be a filter ? It looks like a usecase for a function to me. It does not transform the value, but loads something different.

Reply:
I agree, see #3149